qt/scientific_interfaces/Inelastic/QENSFitting/FunctionBrowser/FitTypes.cpp

```diff
@@ -25,6 +25,8 @@ std::map<IqtTypes::FitType, TemplateSubTypeDescriptor> TemplateSubTypeImpl<IqtTy
     {IqtTypes::FitType::None, {"None", "", {ParamID::NONE, ParamID::NONE}}},
     {IqtTypes::FitType::StretchExponential,
      {"Stretch Exponential", "StretchExp", {ParamID::STRETCH_HEIGHT, ParamID::STRETCH_STRETCHING}}},
+    {IqtTypes::FitType::TeixeiraWaterIqt,
+     {"Teixeira Water Iqt", "TeixeiraWaterIqt", {ParamID::TWI_AMPLITUDE, ParamID::TWI_GAMMA}}},
```

Comment:
This is what I initially thought, but I believe it takes an index range of the ParamID's. So because `ParamID::TWI_TAU` is defined in the enum inbetween `ParamID::TWI_AMPLITUDE` and `ParamID::TWI_GAMMA`, it is included.